### DISCUSSION TITLE
I need to train new "continuous" digital images

### DISCUSSION THREAD

**Discussion by bucklevision** [jomjol/AI-on-the-edge-device#2311] [General]
I am using a digital meter with slightly odd digits but also some irritating reflections that I cannot remove. I'm trying to use extended resultion to get decimal values from the rotating digits as with the "cont" model. Therefore I need to try to re-train the cont model, however the documentation around how to do this this seems to be only partially helpful, and it also seems to suggest that for cont training, you need to fork and use the analogue model? Is this actually correct? I could do with more handholding regarding the process!

**Comment by ghost** [comment]
Could you post pictures of your digital meter?

As far as I know you can use a higher resolution on input. The edge-AI-on-device reads the size from the model and resize the pictures. But for digits it's normally not necessary.

https://github.com/haverland/Tenth-of-step-of-a-meter-digit is the project for dig-class100. 

Josefs dig-cont is not open currently for digits.

Best you use the https://github.com/haverland/Tenth-of-step-of-a-meter-digit/blob/master/dig-class100-s2_bw.ipynb.   

set the input_shape for instance to (64,40,3) for rgb in the first section and run the model.
```
input_shape = (32, 20,1)            # will be calculated
```

The images you can collect with https://github.com/haverland/collectmeterdigits

**Reply by ghost** [reply]
Do not use grayscale. It is not supported by edge-AI-on-device.
